apps/i18n/common/en_us.json

```diff
@@ -1887,6 +1887,7 @@
   "resetProgram": "Reset",
   "resetPassword": "Reset password",
   "resetTeacherPasswordTooltip": "You cannot reset passwords for teacher accounts in your section.",
```

Comment:
Is this string still used anywhere or can it be removed since the buttons are now disabled or removed?

Reply:
Oh I see your own conversation lol.